src/integrate/scarches/config.vsh.yaml

```diff
@@ -0,0 +1,140 @@
+functionality:
+  name: scarches
+  namespace: "integrate"
+  version: "dev"
+  description: "Performs reference mapping with scArches"
+  authors:
+    - name: Vladimir Shitov
+      email: vladimir.shitov@helmholtz-muenchen.de
+      roles: [ author ]
+      props: { github: vladimirshitov, orcid: "0000-0002-1960-8812" }
+  argument_groups:
+    - name: Inputs
+      arguments:
+        - name: "--input"
+          alternatives: ["-i"]
+          type: file
+          description: Input h5mu file to use as a query
+          direction: input
+          required: true
+        - name: "--modality"
+          type: string
+          default: "rna"
+          required: false
+        - name: "--reference"
+          choices: ["HLCA"]
+          alternatives: ["-r"]
+          type: string
+          description: Name of the reference dataset
+          required: false
+        - name: "--unlabeled_category"
+          type: string
+          description: "Value used for unlabeled cells"
+          default: "Unknown"
+          required: false
+        - name: "--labels_key"
+          type: string
+          description: "Name of the column containing cell types"
+          default: "leiden"
+          required: false
+        - name: "--predicted_labels_key"
+          type: string
+          description: Name of the column that will contain cell types predicted by scANVI
```

Comment:
don't use scANVI for this!
XGboost is better

Reply:
check code from czi annotation

Reply:
Also, should I simply add code that assumes that we have a classifier? Or is it better to add components for building classifiers and then for running them?

Reply:
good point... do we want to separate ref mapping from label transfer? I think mostly this is sth we want to do together, no? If we use an HLCA ref model, we should expect trained classifiers for that already i think.

Reply:
Yes, but what if we want to use several classifiers or if there will be new classifiers available with better quality? With a separate component we can easily apply them to the mapped data. With one component we will need to run mapping each time

Reply:
true... either we run all classifiers for one model if we ref map, or we do this separately. I'm okay with both. If it's a separate component we have file I/O to worry about.